services/workflows-service/src/global.d.ts

```diff
@@ -2,6 +2,7 @@ declare module '@prisma/client' {
   import type {
     WorkflowRuntimeData as _WorkflowRuntimeData,
     WorkflowDefinition as _WorkflowDefinition,
+    Customer as _Customer,
```

Comment:
`_` is our convention for unused. Until we vote on it in the past we've used a `_` suffix.
```suggestion
    Customer as Customer_,
```